client/login/controller.js

```diff
@@ -29,6 +29,8 @@ const enhanceContextWithLogin = context => {
 			locale={ lang }
 			path={ path }
 			twoFactorAuthType={ twoFactorAuthType }
+			socialService={ socialService }
+			socialServiceResponse={ context.hash }
```

Comment:
What does `context.hash` contain exactly? Is that the result of a hash function applied to the `context` object? If so, do you know where this magic happens? Also in that case, `socialServiceResponse` doesn't seem like a good name.

Reply:
Thanks a ton for the clarification!

> Maybe we should just pass down the information that we're on a callback url, but I found this approach more generic as other social services probably won't do that automatically for us.

I tend not to optimize for the future, and especially given our new focus, I would probably drop anything that isn't tightly related to Google for now.